EIPS/eip-7668.md

```diff
@@ -12,23 +12,23 @@ created: 2024-03-31
 
 ## Abstract
 
-Require the bloom filters in an execution block, including at the top level and in the receipt object, to be empty.
+Require the bloom filters in an execution block, including at the top level and in the receipt object, to be hardcoded to `0xffff...ff`.
 
 ## Motivation
 
 Logs were originally introduced to give applications a way to record information about onchain events, which decentralized applications (dapps) would be able to easily query. Using bloom filters, dapps would be able to quickly go through the history, identify the few blocks that contained logs relative to their application, and then quickly identify which individual transactions have the logs that they need.
 
-In practice, this mechanism is far too slow. Almost all dapps that access history end up doing so not through RPC calls to an Ethereum node (even a remote-hosted one), but through centralized extra-protocol services.
+In practice, this mechanism is far too slow. Almost all dapps that access history end up doing so not through RPC calls to an Ethereum node (even a remote-hosted one), but through centralized extra-protocol services.  Not too mention, in a chain with relatively full blocks, the false positive rate in bloom filters has increased to the point of making bloom filters almost useless.
```

Comment:
```suggestion
In practice, this mechanism is far too slow. Almost all dapps that access history end up doing so not through RPC calls to an Ethereum node (even a remote-hosted one), but through centralized extra-protocol services.  Not to mention, in a chain with relatively full blocks, the false positive rate in bloom filters has increased to the point of making bloom filters almost useless.
```

One could also point to the fact that the gas limit will likely rise to 60M, which means more txs are packed in the block and which would thus bloat the filter even more because of more gas available for LOGs.